closed/OpenJ9.gmk

```diff
@@ -257,13 +327,19 @@ run-preprocessors-j9 : stage-j9 \
 build-j9 : run-preprocessors-j9
 	@$(ECHO) Compiling OpenJ9 in $(OUTPUT_ROOT)/vm
 	(export OPENJ9_BUILD=true $(EXPORT_NO_USE_MINGW) && cd $(OUTPUT_ROOT)/vm && $(MAKE) $(MAKEFLAGS) all)
-	@$(ECHO) OpenJ9 compile complete
-	# jvm and jsig are required for compiling other java.base support natives
+	@$(ECHO) OpenJ9 compile completei
+
+	@$(CP) -p $(OUTPUT_ROOT)/vm/java*.properties $(OUTPUT_ROOT)/support/modules_libs/java.base
+	@$(CP) -p $(OUTPUT_ROOT)/vm/J9TraceFormat.dat $(OUTPUT_ROOT)/support/modules_libs/java.base
+	@$(CP) -p $(OUTPUT_ROOT)/vm/OMRTraceFormat.dat $(OUTPUT_ROOT)/support/modules_libs/java.base
+	@$(CP) -p $(OUTPUT_ROOT)/vm/options.default $(OUTPUT_ROOT)/support/modules_libs/java.base
+	
```

Comment:
There are macros for these files.  A copy like this is going to happen every time, regardless if the content is new or not.

Reply:
Another copy macro is SetupCopyFiles (see /openj9-openjdk-jdk9/make/common/MakeBase.gmk):

e.g.

```
 $(eval $(call SetupCopyFiles, COPY_LIBS, \
        SRC := $(LIBS_DIR), \
        DEST := $(JDK_OUTPUTDIR)/lib, \
        FILES := $(filter %$(SHARED_LIBRARY_SUFFIX), $(call CacheFind, $(LIBS_DIR))), \
    ))
TARGETS += $(COPY_LIBS) 
```

Reply:
I couldn't put these files where I do the the other copies (using the macro) for java.base (generated_target_rules_build) as those files are already copied to the build jdk image in that recipe and make doesn't like the same files having 2 targets. It can't be put into the other recipe (generated_target_rules) as it needs to be in java.base before this recipe is called as jlink is used to partially build the image jdk and jre before this is called.